notifier.rst

```diff
@@ -375,22 +375,23 @@ both an email and sms notification to the user.
 The default notification also has a ``content()`` and ``emoji()`` method to
 set the notification content and icon.
 
-Symfony provides three types of recipients:
+Symfony provides two types of recipients:
 
 :class:`Symfony\\Component\\Notifier\\Recipient\\NoRecipient`
     This is the default and is useful when there is no need to have
     information about the receiver. For example, the browser channel uses
     the current requests's :ref:`session flashbag <flash-messages>`;
 
 :class:`Symfony\\Component\\Notifier\\Recipient\\Recipient`
-    This contains only the email address of the user and can be used for
-    messages on the email and browser channel;
-
-:class:`Symfony\\Component\\Notifier\\Recipient\\AdminRecipient`
     This can contain both email address and phonenumber of the user. This
     recipient can be used for all channels (depending on whether they are
     actually set).
 
+.. versionadded:: 5.2
+
+    The `AdminRecipient` class was removed in Symfony 5.2, you should use
```

Comment:
As this is a bc break, do we usually mark this as such?

Reply:
I think it's fine like this (except that you need double backticks to mark inline code in reStructured Text)

Reply:
done